rsearch/.gitignore

```diff
@@ -0,0 +1 @@
+vendor/
```

Comment:
I maintain godep directory, but with VENDOREXPERIMENT `vendor/` folder gets created automatically. This line is to prevent accidental commits including this folder.